lib/node_modules/@stdlib/math/base/special/ceil2/package.json

```
@@ -64,5 +64,84 @@
     "prevpow2",
     "nearest",
     "number"
-  ]
+  ],
+  "__stdlib__": {
+    "scaffold": {
+      "$schema": "math/base@v1.0",
+      "base_alias": "ceil2",
+      "alias": "ceil2",
+      "pkg_desc": "round a double-precision floating-point number to the nearest power of two toward positive infinity",
+      "desc": "rounds a double-precision floating-point number to the nearest power of two toward positive infinity",
+      "short_desc": "ceil to power of two",
```

```suggestion
      "short_desc": "",
```

No good short description here. Will need to add a custom description in the scaffolding.